### PR TITLE
feat(one-off): allow attached one-off to be run asynchronously

### PR DESCRIPTION
**Pull request by ipfaze**
fix: #313 
- [ ] Add a [changelog entry](https://changelog.scalingo.com/)
=> n/a

### CODE REVIEW

File: run.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 
 	"gopkg.in/errgo.v1"
 
-	"github.com/Scalingo/go-scalingo/v6/http"
+	httppkg "github.com/Scalingo/go-scalingo/v6/http"
```
Comment (EtienneM) [inline run.go:10]: Why the package renaming? I don't see any conflict in package imports
```suggestion
	"github.com/Scalingo/go-scalingo/v6/http"
```

Comment (ipfaze) [reply run.go:10]: I tried something and didn't revert this renaming change :disappointed:

File: run.go [hunk 2]
```diff
@@ -22,12 +22,14 @@ type RunOpts struct {
...
+	Async      bool
 	HasUploads bool
 }
 
 type RunRes struct {
-	Container *Container `json:"container"`
-	AttachURL string     `json:"attach_url"`
+	Container    *Container `json:"container"`
+	AttachURL    string     `json:"attach_url"`
+	OperationURL string     `json:"operation_url"`
```
Comment (EtienneM) [inline run.go:32]: There is no need for a JSON tag here I think. We don't JSON decode the operation URL.

```suggestion
	OperationURL string     `json:"-"`
```

File: run.go
Comment (EtienneM) [inline run.go:1]: While you are on this file, it could be nice to also replace the use of `errgo.Mask` in this file.

**Review by EtienneM — APPROVED** [review]
just a comment but LGTM

File: run.go
Comment (ipfaze) [reply run.go:1]: I'll do it, thanks for the reminder 😉

**Review by SCedricThomas — APPROVED** [review]

**Pull request merged**